src/main/java/com/onlinebookstore/service/impl/UserServiceImpl.java

```diff
@@ -24,6 +26,7 @@ public UserResponseDto register(UserRegistrationRequestDto requestDto)
                     + requestDto.getEmail());
         }
         User user = userMapper.toModel(requestDto);
+        user.setPassword(passwordEncoder.encode(requestDto.getPassword()));
```

Comment:
find role user and set it up
```suggestion
        user.setPassword(passwordEncoder.encode(requestDto.getPassword()));
        user.setRoles(Set.of(role));
```